skyportal/handlers/api/spectrum.py

```diff
@@ -22,7 +22,9 @@ def post(self):
                         type: array
                         items:
                           type: integer
-                        description: Group IDs that spectrum will be associated with
+                        description: |
+                          Group IDs that spectrum will be associated with. If 'all',
+                          will be shared with all groups associated source is visible to.
```

Comment:
all should share it to everyone on the site. Maybe we can implement using the sitewide_group?

Reply:
Done